packages/forms/src/widgets/TogglePasswordWidget/TogglePasswordWidget.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import TogglePasswordWidget from './TogglePasswordWidget';
+
+describe('TogglePasswordWidget', () => {
```

Comment:
please add the test checking if interaction work correctly, check that a t first is renderered a password field, simulate click and check that it is now renderered as an input